src/overlays/actors/ovl_En_Sth/z_en_sth.h

```diff
@@ -5,9 +5,31 @@
 
 struct EnSth;
 
+typedef void (*EnSthActionFunc)(struct EnSth*, GlobalContext*);
+
+#define ENSTH_GET_F(thisx) ((thisx)->params & 0xF)
+#define ENSTH_GET_100(thisx) ((thisx)->params & 0x100)
+
+enum {
+    /* 3 */ ENSTH_F_1 = 1,
+    /* 3 */ ENSTH_F_2,
+    /* 3 */ ENSTH_F_3,
+    /* 4 */ ENSTH_F_4,
```

Comment:
```suggestion
    /* 1 */ ENSTH_F_1 = 1,
    /* 2 */ ENSTH_F_2,
    /* 3 */ ENSTH_F_3,
    /* 4 */ ENSTH_F_4,
```
Matching up comment with enum value